src/packages/perry/index.ts

```diff
@@ -34,9 +34,13 @@ import stopListeners from '@/packages/stop-listeners';
 /** Perry stateless notifier */
 import notify from '@/packages/perry-notify';
 
+/** Screen recorder */
+import screenRecorder from '@/packages/record-screen';
```

Comment:
should we name the package as `@/package/screen-recorder` just for the sake of consistency?